<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>497</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>706</width>
    <height>497</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>706</width>
    <height>497</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Guiñote</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
background-color: rgb(0, 85, 0);
}

#formLayout{
background-color: rgb(255,255,255)
}



</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>611</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="leftMargin">
      <number>40</number>
     </property>
     <property name="rightMargin">
      <number>40</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_nombre_usuario">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nombre de usuario:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="text_nombre_usuario">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_contrasenya">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Contraseña:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="text_contrasenya">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="boton_login">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_foto_login">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/logo/150.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="boton_registrarse">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>380</y>
      <width>261</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>30</height>
     </size>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>7</red>
          <green>64</green>
          <blue>229</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>21</red>
          <green>28</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>7</red>
          <green>64</green>
          <blue>229</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>21</red>
          <green>28</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>¿No tienes cuenta? ¡Registrate gratis!</string>
    </property>
    <property name="icon">
     <iconset theme="network-receive">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Imagenes/imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
